src/Adyen/Service/ResourceModel/Account/CheckAccountHolder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Adyen\Service\ResourceModel\Account;
+
+use Adyen\Service;
+
+class CheckAccountHolder  extends \Adyen\Service\AbstractResource
+{
+    /**
+     * @var string
+     */
+    protected $endpoint;
+
+    /**
+     *
+     * @param Service $service
+     */
+    public function __construct($service)
+    {
+        $this->endpoint = $service->getClient()->getConfig()->get('endpointAccount') .
+            '/' . $service->getClient()->getApiAccountVersion() . 'checkAccountHolder';
```

Comment:
Is this missing a forward slash before `checkAccountHolder`?

Reply:
You're right, my bad, just fixed it